charts/gardener-extension-provider-alicloud/values.yaml

```diff
@@ -74,6 +74,8 @@ config:
 #  - image-id1
 #  - image-id2
 #  ...
+  service:
+    backendLoadBalancerType: slb.s1.small
```

Comment:
Shouldn't this be rather configured in the `CloudProfile` instead of in the extension config? 👀

Reply:
It's a good idea, but I have no idea how to leverage configure in `CloudProfile` in web hooks of the provider? Could you please provide one example for reference if possible? :)

Reply:
Hm I understand, I don't see a straight-forward way either. OK, then ignore the question for now. thanks!